app/src/main/java/chat/rocket/android/app/DateTimeHelper.kt

```diff
@@ -1,8 +1,6 @@
 import android.content.Context
 import chat.rocket.android.R
-import org.threeten.bp.LocalDate
-import org.threeten.bp.LocalDateTime
-import org.threeten.bp.Period
+import org.threeten.bp.*
```

Comment:
Interesting... the auto-complete imports all `org.threeten.bp` package. We should avoid it